app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -0,0 +1,277 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useMemo, useRef, useState} from 'react';
+import {defineMessages, useIntl} from 'react-intl';
+import {DeviceEventEmitter, Keyboard, Platform, StyleSheet, View} from 'react-native';
+import {SafeAreaView} from 'react-native-safe-area-context';
+
+import {fetchChannelMemberships} from '@actions/remote/channel';
+import {searchProfiles} from '@actions/remote/user';
+import Search from '@components/search';
+import UserList from '@components/user_list';
+import {Events, General, Screens} from '@constants';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import {debounce} from '@helpers/api/general';
+import useNavButtonPressed from '@hooks/navigation_button_pressed';
+import {t} from '@i18n';
+import {openAsBottomSheet, setButtons} from '@screens/navigation';
+import NavigationStore from '@store/navigation_store';
+import {showRemoveChannelUserSnackbar} from '@utils/snack_bar';
+import {changeOpacity, getKeyboardAppearanceFromTheme} from '@utils/theme';
+import {filterProfilesMatchingTerm} from '@utils/user';
+
+type Props = {
+    canManageMembers: boolean;
+    channelId: string;
+    componentId: string;
+    currentTeamId: string;
+    currentUserId: string;
+    teammateNameDisplay: string;
+    tutorialWatched: boolean;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+    },
+    searchBar: {
+        marginLeft: 12,
+        marginRight: Platform.select({ios: 4, default: 12}),
+        marginVertical: 12,
+    },
+});
+
+const messages = defineMessages({
```

Comment:
Unless there's a reason to use `defineMessages` function, I think we can remove it from all the places being called.  It only add extra function calls.

Reply:
This is a preference I've started using in some places to consolidate messages in one area, outside of the component.